### DISCUSSION TITLE
More Platform Support

### DISCUSSION THREAD

**Discussion by fexhvhbdsf864** [Wohlstand/TheXTech#651] [Ideas]
iOS / Apple TV
PlayStation Portable
PlayStation 2
PlayStation 3
Nintendo DSi
GameCube
Dreamcast
XBOX Series / One
XBOX
XBOX360

**Comment by 60fpshacksrock** [comment]
I second.

**Comment by Wohlstand** [comment]
- For the **iOS / Apple TV** I will need a hardware with iOS (I don't have any right now) to debug the stuff, and then, users will need to side-load the game builds using special tools.
- **PSP** has a small amount of RAM, but in theory the thing could work with strong limitations. There is a chance that here is no enough video memory, but I am not sure.
- **PS2** is same as PSP, has few resources to work. Additionally, the homebrew SDK is needed to get something to work here.
- **PS3** has enough resources to run the game, but again, the homebrew SDK is needed.
- For **Nintendo DSi**, ask @ds-sloth, there are some experiments are going, but, this platform is EXTREMELY low-end, and therefore, it will have a lot of limitations being enabled compared to other platforms.
- **GameCube** has a DevkitPro support, however, this is another low-end platform, it's a bit quicker than DSi, but significantly slower than Wii (which is already supported).
- **Dreamcast** is very similar super-low-end as DSi. The strong limitations are possible.
- Any **XBox** thing needs a homebrew SDK. Original XBox seems enough to run the game, the video chip is questionable, I don't know how much video memory it has.

**Comment by ds-sloth** [comment]
This is a good wishlist. I've converted it to a discussion because we would want to have a single issue per platform. An important note: for platforms that are not trivial to port to (have a fully working SDL2 port and a compiler toolchain supporting CMake), the difficulty of creating and maintaining a port will be very high. We'd want someone to volunteer as a port maintainer for any such target, and I don't think that Wohlstand or I are able to at this point. Please see http://github.com/Wohlstand/TheXTech/wiki/Porting for some of the considerations.

**Comment by ds-sloth** [comment]
For iOS, another option (doesn't require sideloading) is to use the Emscripten port as a web app. This works moderately well currently but isn't quite to production quality yet. One particular problem is that the assets aren't properly cached yet (they get re-downloaded every time the game is started). We can open an additional issue to track the use of the Emscripten port for iOS.
